QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,187 @@
+This document is a declaration of software quality for the `rmw_implementation` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `rmw_implementation` Quality Declaration
+
+The package `rmw_implementation` claims to be in the **Quality Level 4** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 1 in REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+## Version Policy [1]
+
+### Version Scheme [1.i]
+
+`rmw_implementation` uses `semver` according to the recommendation for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#versioning).
+
+### Version Stability [1.ii]
+
+`rmw_implementation` is not yet at a stable version, i.e. >= 1.0.0.
+
+### Public API Declaration [1.iii]
+
+**`rmw_implementation` does not expose a public API.**
+
+### API Stability Within a Released ROS Distribution [1.iv]/[1.vi]
+
+`rmw_implementation` will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released.
+
+### ABI Stability Within a Released ROS Distribution [1.v]/[1.vi]
+
+`rmw_implementation` will maintain ABI stability within a ROS distribution.
+
```

Comment:
I think it's better leaving [v.i] on its own, even if it can be considered repeated information

```suggestion
### API Stability Policy [1.iv]

`rmw_implementation` will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released.

### ABI Stability Policy [1.v]

`rmw_implementation` contains C code and therefore must be concerned with ABI stability, and will maintain ABI stability within a ROS distribution.

### ABI and ABI Stability Within a Released ROS Distribution [1.vi]

`rmw_implementation` will not break API nor ABI within a released ROS distribution, i.e. no major releases once the ROS distribution is released.
```